### PR TITLE
Listing and deleting worklogs for given issue.

### PR DESCRIPTION
**Pull request by mmswiderski**
Listing and deleting worklogs for given issue.
Inspect basic info for given issue key.

Minor refactoring

Signed-off-by: mmswiderski <mmswiderski@gmail.com>

### CODE REVIEW

File: cmd/issue/worklog/worklogDelete.go [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+	"github.com/spf13/viper"
+)
+
+//Cmd workload add command
+var worklogDeleteCmd = &cobra.Command{
+	Use:     "remove ISSUE_KEY",
+	Aliases: []string{"r"},
+	Args:    cobra.MinimumNArgs(1),
+	Short:   "Delete all worklogs for logged user from provided ISSUE_KEY",
+	Run: func(cmd *cobra.Command, args []string) {
```
Comment (sotomskir) [inline cmd/issue/worklog/worklogDelete.go]: There is a lot of logic in this command. I think it should be moved to a function in jiraApi.go

File: cmd/issue/worklog/worklogList.go [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+	"github.com/spf13/cobra"
+	"os"
+	"strconv"
+)
+
+//Cmd workload add command
+var worklogListCmd = &cobra.Command{
+	Use:     "list ISSUE_KEY",
+	Aliases: []string{"l"},
+	Args:    cobra.MinimumNArgs(1),
```
Comment (sotomskir) [inline cmd/issue/worklog/worklogList.go]: cobra.ExactArgs(1)

File: cmd/issue/worklog/worklogList.go [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+	"github.com/sotomskir/jira-cli/jiraApi"
+	"github.com/spf13/cobra"
+	"os"
+	"strconv"
+)
+
+//Cmd workload add command
+var worklogListCmd = &cobra.Command{
+	Use:     "list ISSUE_KEY",
+	Aliases: []string{"l"},
```
Comment (sotomskir) [inline cmd/issue/worklog/worklogList.go]: Please add ls alias to be consistent with other commands

File: cmd/issue/worklog/worklogList.go [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+	Args:    cobra.MinimumNArgs(1),
+	Short:   "List worklog for given task",
+	Run: func(cmd *cobra.Command, args []string) {
+		key := args[0]
+		resp, err := jiraApi.ListWorklog(key)
+		if err != nil {
+			logrus.Errorf("There was an error while listing worklogs for issue %s.", key)
+			return
```
Comment (sotomskir) [inline cmd/issue/worklog/worklogList.go]: os.Exit(1)

File: jiraApi/jiraApi.go [hunk 5]
```diff
@@ -188,6 +188,20 @@ func Worklog(key string, min uint64, com string) error {
...
 
+//ListWorklog for specified JIRa issue
+func ListWorklog(key string) (worklogs models.WorklogList, error error) {
+	logrus.Infof("Attempting to list worklogs for issue %s.", key)
+	response := models.WorklogList{}
+	_, err := execute(resty.MethodGet, fmt.Sprintf("rest/api/2/issue/%s/worklog", key), nil, &response)
+	return response, err
+}
+
+func DeleteWorklog(key string, id string) (status int, error error) {
```
Comment (sotomskir) [inline jiraApi/jiraApi.go]: exported method should have comment

File: cmd/issue/issueInspect.go [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+	"github.com/sirupsen/logrus"
+	"github.com/sotomskir/jira-cli/jiraApi"
+	"github.com/spf13/cobra"
+	"os"
+)
+
+// Cmd represents the issue command
+var inspectCmd = &cobra.Command{
+	Use:     "inspect",
+	Aliases: []string{"ins"},
```
[the remark this answers is not in the document]
Comment (mmswiderski) [reply cmd/issue/issueInspect.go]: I was thinking that double 'i' might be misleading - `jira-cli i i XX-XXX`
Done.

[the remark this answers is not in the document]
Comment (mmswiderski) [reply cmd/issue/issueInspect.go]: I was thinking that double 'i' might be misleading - jira-cli i i XX-XXX
Done.

File: cmd/issue/issueInspect.go [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+	"github.com/olekukonko/tablewriter"
+	"github.com/sirupsen/logrus"
+	"github.com/sotomskir/jira-cli/jiraApi"
+	"github.com/spf13/cobra"
+	"os"
+)
+
+// Cmd represents the issue command
+var inspectCmd = &cobra.Command{
+	Use:     "inspect",
```
[the remark this answers is not in the document]
Comment (mmswiderski) [reply cmd/issue/issueInspect.go]: Done.

File: cmd/issue/issueInspect.go [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+	Aliases: []string{"ins"},
+	Args:    cobra.MinimumNArgs(1),
```
[the remark this answers is not in the document]
Comment (mmswiderski) [reply cmd/issue/issueInspect.go]: Done.

File: cmd/issue/issueInspect.go [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+	Short:   "Fetch data for given issue.",
+	Run: func(cmd *cobra.Command, args []string) {
+		key := args[0]
+		issue, err := jiraApi.GetIssue(key)
+		if err != nil {
+			logrus.Errorf("Selected issue %s does not exist.", key)
+			return
```
[the remark this answers is not in the document]
Comment (mmswiderski) [reply cmd/issue/issueInspect.go]: Done.

File: cmd/issue/worklog/worklogDelete.go [hunk 10]
```diff
@@ -0,0 +1,70 @@
...
+		key := args[0]
+		user := viper.GetString("JIRA_USER")
+		resp, err := jiraApi.ListWorklog(key)
+		sum := 0
+		sumErr := 0
+
+		if err != nil {
+			logrus.Errorf("There was an error while listing worklogs for issue %s.", key)
+			return
```
[the remark this answers is not in the document]
Comment (mmswiderski) [reply cmd/issue/worklog/worklogDelete.go]: Done.

File: cmd/issue/worklog/worklogDelete.go [hunk 11]
```diff
@@ -0,0 +1,70 @@
...
+	"github.com/sotomskir/jira-cli/jiraApi"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+//Cmd workload add command
+var worklogDeleteCmd = &cobra.Command{
+	Use:     "remove ISSUE_KEY",
+	Aliases: []string{"r"},
+	Args:    cobra.MinimumNArgs(1),
```
[the remark this answers is not in the document]
Comment (mmswiderski) [reply cmd/issue/worklog/worklogDelete.go]: Done.

File: cmd/issue/worklog/worklogDelete.go (hunk 1 above)
Comment (mmswiderski) [reply cmd/issue/worklog/worklogDelete.go]: Done. Refactored. Tested.

File: cmd/issue/worklog/worklogList.go (hunk 2 above)
Comment (mmswiderski) [reply cmd/issue/worklog/worklogList.go]: Done.

File: cmd/issue/worklog/worklogList.go (hunk 3 above)
Comment (mmswiderski) [reply cmd/issue/worklog/worklogList.go]: Done.

File: cmd/issue/worklog/worklogList.go (hunk 4 above)
Comment (mmswiderski) [reply cmd/issue/worklog/worklogList.go]: Done.

File: jiraApi/jiraApi.go (hunk 5 above)
Comment (mmswiderski) [reply jiraApi/jiraApi.go]: Done.

**Pull request merged**